packages/@o3r/core/schematics/rule-factories/generic-updates/index.ts

```diff
@@ -0,0 +1,27 @@
+import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+
+/**
+ * Generic basic updates to be done on a repository
+ */
+export function genericUpdates(): Rule {
+
+  const updateGitIgnore = (tree: Tree, _context: SchematicContext) => {
+    // update gitignore
+    if (tree.exists('/.gitignore')) {
+      let gitignore = tree.readText('/.gitignore');
+
+      gitignore = gitignore.replace(/(\r\n|\n)\/node_modules(\r\n|\n)/, '$1node_modules$2');
```

Comment:
why not `/^\/node_modules$/` ?

Reply:
`tree.readText` is giving a single string as output, replacing the new lines.

Here is a small snapshot:
`# See http://help.github.com/ignore-files/ for more about ignoring files.\r\n\r\n# Compiled output\r\n/dist\r\n/tmp\r\n/out-tsc\r\n/bazel-out\r\n\r\n# Node\r\n/node_modules\r\nnpm-debug.log\r\nyarn-error.log\r\n\r\n# IDEs`